### DISCUSSION TITLE
When will the Docker image be updated?

### DISCUSSION THREAD

**Discussion by joanjgm** [CorentinTh/it-tools#1317] [General]
When will the Docker image be updated? I still don't see the Regex new tools on the docker image

**Comment by Tyree** [comment]
246 PRs and no releases since May. I wonder if this project is still kicking.

**Comment by goalkeepr** [comment]
I'm starting to wonder. Maybe someone should fork it and keep going!

**Comment by sharevb** [comment]
Hi @goalkeepr , @Tyree , unfortunately @CorentinTh said, he has a lot of other projects, and have little time to merge (he merged some of my/other pr), and seems not to want to share maintener status. I personaly made more than hundred PR, I wonder if I will not fork if this situation remains (unanswered) for again sometime

**Comment by CorentinTh** [comment]
Hi everyone,

Thank you all for your patience and understanding regarding the project. First off, I sincerely apologize for the delays, I've been a bit overwhelmed with the influx of contributions and issues and keeping up with everything.

I'll be on holiday starting next week, which will give me some much-needed time to focus on IT-Tools. My plan is to implement some structural changes to make it easier for everyone to contribute and maintain the project going forward. I also have plans for the next major version, but first I need to sort out the current state 

I genuinely care about IT-Tools, and I’m committed to ensuring it continues to thrive.

(I'll publish a release on Monday for the current contributions)

**Reply by Tyree** [reply]
Thanks for the update, @CorentinTh !
I hope you take heart in the fact that the only reason people are clamoring for updates and contributing all the PRs is because you've created something awesome. 😊

**Comment by sharevb** [comment]
Hi, for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Comment by goalkeepr** [comment]
Thank you, sir!
